controller/rest/sigstore.go

```diff
@@ -400,10 +401,8 @@ func handlerSigstoreVerifierGetAll(w http.ResponseWriter, r *http.Request, ps ht
 		return
 	}
 
-	resp := make(map[string]api.REST_SigstoreVerifier, len(verifiers))
-	for key, verifier := range verifiers {
-		resp[key] = CLUSVerifierToRESTVerifier(verifier)
-	}
+	resp := make(map[string]interface{})
```

Comment:
The return object's type should be defined in apis.go & described in apis.yaml instead of simply an interface type in the api handler